src/modal/modal.stories.ts

```diff
@@ -13,7 +13,7 @@ import { Modal, ModalService } from "../";
 	selector: "app-sample-modal",
 	template: `
 		<ibm-modal>
-			<ibm-modal-header (closeSelect)="closeModal()">Header text</ibm-modal-header>
+			<ibm-modal-header (closeSelect)="closeModal()">header label</ibm-modal-header>
```

Comment:
@cal-smith Suggested the name and it goes very well with Carbon's Modal naming.

Reply:
Yeah, label is fine, I was just trying to point out the lowercase to uppercase `H` :)
Sorry for being a bit ambiguous.

Reply:
Can we get a capital H in `header label` so it would be `Header label`, pretty please? :)

You can even name it `Modal heading` to match vanilla and React flavors.